backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/payment_allocation/InvoiceRow.java

```diff
@@ -61,6 +61,10 @@
 	@Getter
 	private final String documentNo;
 
+	@ViewColumn(seqNo = 20, widgetType = DocumentFieldWidgetType.Text, widgetSize = WidgetSize.Small, captionKey = "POReference")
```

Comment:
```suggestion
	@ViewColumn(seqNo = 25, widgetType = DocumentFieldWidgetType.Text, widgetSize = WidgetSize.Small, captionKey = "POReference")
```

..don't want to have seqNo=20 twice